### ISSUE TITLE
Optional variables

### ISSUE THREAD

**Issue by Walkman100** [deavmi/Picy#34]
would it be a good idea to have some variables (e.g. `$showBuiltWithPicy`) not HAVE to be set, and have a default value? (in this case true)

**Comment by deavmi** [comment]
@Walkman100 No. Let me think about this tomorrow. It's night and the logic is not in my head currently.
